extensions/podman/src/extension.ts

```diff
@@ -55,9 +66,21 @@ type MachineInfo = {
   diskSize: number;
 };
 
+interface PodmanInfo {
+  podmanVersion: string;
+  lastUpdateCheck: number;
+}
+
+function getPodman(): string {
+  if (isWindows) {
+    return 'c:\\Program Files\\RedHat\\Podman\\podman.exe';
+  }
+  return 'podman';
```

Comment:
we might wanna enforce a known location on `mac` too. WDYT?

Reply:
On macOS the install location is `/Applications/podman/podman` for the client binary